dev/com.ibm.ws.jmx/src/com/ibm/ws/jmx/internal/DelayedMBeanActivatorHelper.java

```diff
@@ -1,13 +1,14 @@
-/*******************************************************************************
- * Copyright (c) 2011 IBM Corporation and others.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
+/*
+ * IBM Confidential
```

Comment:
Same issue with the license header here too.

Reply:
I reverted my changes that I had orignally introduced into DelayedMBeanActivatorHelper.java such that it should now reflect what had been there before, w the same license header info.